rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
```

Comment:
Ah ok! At least for the benchmark-suite, I was wondering if it is possible to remove `path` so that someone who just only has the ruby-bench-suite repo can just run the benchmarks without having to clone `rails`. Not important for now, I can work on it after we merge it.